### DISCUSSION TITLE
Lora trigger words

### DISCUSSION THREAD

**Discussion by crapthings** [huggingface/diffusers#7855] [Q&A]
How do i trigger lora effect after load?

what is different between fuse and not fused?

**Comment by tolgacangoz** [comment]
You can see several examples of how to use triggers [in the docs page](https://huggingface.co/docs/diffusers/main/en/tutorials/using_peft_for_inference). The creator of LoRA usually says what the trigger word(s) is/are, e.g., look at [this LoRA](https://huggingface.co/goofyai/3d_render_style_xl#tips). Or sometimes, [there is no need to use a trigger word](https://huggingface.co/ostris/ikea-instructions-lora-sdxl).
You can examine fusing [on this page at docs](https://huggingface.co/docs/diffusers/main/en/using-diffusers/merge_loras#fuselora).
If you have any further questions after reviewing these materials, please don't hesitate to ask.

**Comment by crapthings** [comment]
@tolgacangoz 

it looks it must use token even after using fuse lora
so the token must trained and use the token to inference right?

```
pipe.set_adapters(['cloth', 'face', 'style'], adapter_weights=[0.5, 0.5, 0.5])
cross_attention_kwargs = { 'scale': 1.0  }
```
so this will not trigger lora effect?

**Reply by tolgacangoz** [reply]
This is from the last material I shared before:
>It is important to make the `lora_scale` adjustments in the [`fuse_lora()`](https://huggingface.co/docs/diffusers/main/en/api/loaders/lora#diffusers.loaders.LoraLoaderMixin.fuse_lora) method because it won’t work if you try to pass `scale` to the `cross_attention_kwargs` in the pipeline.
`pipeline.fuse_lora(adapter_names=["ikea", "feng"], lora_scale=1.0)`

Is this what you are looking for?

(I haven't been available enough in the last few weeks, sorry for the late reply.)
